packages/forms/src/lib/search-filter/README.md

```diff
@@ -0,0 +1,83 @@
+# @acpaas-ui/ngx-components/forms
+
+The search-filter is a JIRA-like filter component with remote search capabilities.
+
+## Usage
+
+```typescript
+import { SearchFilterModule } from '@acpaas-ui/ngx-components/search-filter'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() id: string;` | - | Field id. |
+| `@Input() name: string;` | - | Field name. |
+| `@Input() flyoutSize: enum;` | `FlyoutSize.Small` | Passed along to `auiFlyout`. |
+| `@Input() flyoutAlign: string;` | `'left'` | Passed along to `auiFlyout`. |
+| `@Input() label: string;` | `'Filter'` | Field label. |
+| `@Input() labelDeselect: string;` | `'Alles deselecteren'` | Clear button text. |
+| `@Input() labelResults: string;` | `'Resultaten'` | Result list header text. |
+| `@Input() labelNoResults: string;` | `'Geen resultaten gevonden.'` | Text shown when no results are found. |
+| `@Input() choices: SearchFilterChoice[];` | - | Available choices. |
+| `@Input() remote: boolean;` | `false` | Enable remote searching. |
+| `@Input() placeholder: string;` | `'Zoeken'` | Search field placeholder text. |
+| `@Input() inputDelay: string;` | `'150'` | Delay the search callback on the input field. |
```

Comment:
Is this really a string? Then we should list it in our technical debt document.

Reply:
no number when verifying in the new components.  Older readme had a wrong type defined.